nengo_loihi/tests/test_learning.py

```diff
@@ -224,6 +224,47 @@ def test_multiple_pes(init_function, allclose, plt, seed, Simulator):
                         atol=0.05, rtol=0.05), "Target %d not close" % i
 
 
+def test_pes_deterministic(request, Simulator, seed, plt, allclose):
```

Comment:
`plt` isn't used in this test and should be removed to avoid the empty plot that's generated.